### ISSUE TITLE
Can't get attribute 'C3' on <module 'models.common' from ......

### ISSUE THREAD

**Issue by mkashifraja22** [muhk01/Yolov5-on-Flask#2]
whats the issue

**Comment by huangrz2001** [comment]
did you solve it?

**Comment by kelvinfuhao** [comment]
did you solve it?

**Comment by neso613** [comment]
> did you solve it?

having same issue?

**Comment by stan-maze** [comment]
> > did you solve it?
> 
> having same issue?
yes, and I didn't solve it. I rewrote my own model and changed the way "camera" is used, and the "camera" data flow is actually fine.

**Comment by huangrz2001** [comment]
> > did you solve it?
> 
> having same issue?

yes, and I didn't solve it. I rewrote my own model and changed the way "camera" is used, and the "camera" data flow is actually fine.
